src/index.html

```diff
@@ -16,17 +16,42 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
+  <body class="page">
```

Comment:
There should not be multiple blank lines between elements (lines 20-30). According to the checklist, adding empty lines between multiline sibling blocks is recommended, but this does not apply to spaces between a parent element and its first child.